nrt/data/IEEE14/IEEE14_WithAutomata/description/IEEE14_WithAutomata.tex

```diff
@@ -446,6 +447,79 @@ \subsection{Results}
 
 We observe that changing the tap helps restoring the load terminal voltage but has a negative impact on the network voltage: each time the tap is increased, the network voltage is decreased a little. If many transformers are changing taps at the same time and on the same network area it could lead to a general voltage collapse. In order to avoid this phenomena, other automata called "Tap-Changer Lock" monitor the network voltage and can stop the transformers from changing their taps.
 
+
+\newpage
+\section{Tap Changer (CPP)}
+\label{TapChangerAutomatonCpp}
+
+The tap-changer is an automaton that changes the tap of a transformer if its monitored terminal voltage goes out of a certain range for a certain amount of time. In this test, the default C++ model is tested.
+
+\subsection{Initial Conditions}
+
+Initial conditions are the same that in the IEEE 14-bus basic test cases.
+
+\subsection{Models}
+
+Transformer 5-6 is equipped with a ratio tap-changer that controls the voltage at its higher terminal (node 5). The role of the tap-changer is to maintain the high terminal voltage within a certain range, independently of what happens on the other terminal. \\
```

Comment:
In the other test cases, the ratio tap-changer is there but it is inactive (regulating = false).

Reply:
Setting regulating = true is not enough to activate the tap-changer, we have to add the following line:
`<iidm:terminalRef id="_BUS____5-BUS____6-1_PT" side="ONE"/>`